src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,22 @@
-const humanCatDogYears = number => {};
+const humanCatDogYears = number => {
+  const humanYears = [number];
```

Comment:
Although this is a good naming for what the array represents at this moment, you add the cat and dog years to it later which then makes the naming not represent all the information it is containing. I would be tempted to call it finalArray or result as that is it's intended purpose for this function.